content/docs/12-enterprise-version/Helm Chart Install References

```diff
@@ -0,0 +1,161 @@
+| `env.rootDomain` | "" | String | Specifies the default DNS record mapped to the ingress-nginx-controller load balancer. It is mandatory if false; otherwise, leave it empty. | Required|
+| `env.installerMode` | `self-hosted` | String | Installer mode specified, do not modify the value.| Required|
+| `env.installerCloud` | "" | String | Cloud platform unspecified, leave empty. | Optional |
+
+
+
+### Cluster parameters
+
+Cluster parameters are used to specify the Kubernetes cluster configuration.
+
+
+| **Parameters** | **Default value** | **Type** | **Description** | **Required/Optional** |
+| --- | --- | --- | --- | --- |
+| `stableEndpointAccess` | `False` | Boolean | Specifies whether EKS cluster is deployed in private network access through a stable private endpoint, if true, through a public endpoint. | Optional|
+
+## Registry and SCAR parameters
+
+**Registry and SCAR (Spectro Cloud Admission Registrar)** parameters are mandatory parameters in the **values.yaml** for installing **Palette** using Helm Chart.
+
+| **Parameters**       | **Type**   | **Description**                                             | **Required/Optional** |
+| ----------------- | ------ | -------------------------------------------------------- | ------------------ |
+| `registry.endpoint` | String | The endpoint URL for the registry.                      | Required           |
+| `registry.name`     | String | The name of the registry.                                | Required           |
+| `registry.password` | String | The password for the registry.                           | Required           |
+| `registry.username` | String | The username for the registry.                           | Required           |
+| `scar.endpoint`     | String | The endpoint URL of the Spectro Cloud Admission Registrar (SCAR). | Required           |
+| `scar.username`     | String | The username for the SCAR.                               | Required           |
+| `scar.password`     | String | The password for the SCAR.                               | Required           |
+
+To learn about the default values of the parameters, please contact Spectro Cloud sales.
+
+## Network Address Translation(NATS) parameters
+
+This section explains the parameters for the configuration file containing settings related to the **Network Address Translation (NATS)** messaging system. 
+
+| **Parameters **  | **Default Value** | **Type **   | **Description**                                                                                                               | **Required/Optional** |
+| ------------ | ------------- | ------- | ------------------------------------------------------------------------------------------------------------------------- | ------------------ |
+| `internal`   | `true`        | Boolean | Specifies if NATS shares the ingress load balancer or uses hostNetwork if true otherwise with a cloud load balancer.      | Optional           |
```

Comment:
This description is confusing. Try again and make it clearer what happens for each value.